open_humans/templates/member/member-detail-content.html

```diff
@@ -30,7 +30,7 @@
             data-placement="ottom" class="btn btn-xs btn-info popover-dismiss"
             data-content="
               You must
-              <a href='{% url 'my-member-send-confirmation-email' %}?next={{ request.path }}'>
+              <a href='{% url 'my-member-send-confirmation-email' %}'>
```

Comment:
This isn't a login/signup process so I don't think the `next` should be messed with?